contrib/TeXShop/auto-configure.command

```diff
@@ -61,13 +61,25 @@ defaults write "$TeXShopDir/Contents/Info.plist" CFBundleDocumentTypes -array-ad
 
 echo "Adding Gregorio file extensions to appropriate preference lists"
 #enable syntax coloring and the Typeset button for gabc files
-defaults write TeXShop OtherTeXExtensions -array-add "gabc"
-defaults write TeXShop OtherTeXExtensions -array-add "gtex"
+TeXExtensions=$(defaults read TeXShop OtherTeXExtensions 2>/dev/null)
+if [[ ! $TeXExtensions == *"gabc"* ]]; then
```

Comment:
Why are you quoting "gabc" on this line (and similarly on the if statements below).